tests/foreman/api/test_discoveryrule.py

```diff
@@ -23,221 +24,91 @@
 
 from robottelo.datafactory import valid_data_list
 from robottelo.decorators import tier1
-from robottelo.decorators import tier2
 from robottelo.test import APITestCase
 
 
 class DiscoveryRuleTestCase(APITestCase):
     """Tests for ``katello/api/v2/discovery_rules``."""
 
-    @classmethod
-    def setUpClass(cls):
-        """Create a hostgroup which can be re-used in tests."""
-        super(DiscoveryRuleTestCase, cls).setUpClass()
-        cls.hostgroup = entities.HostGroup().create()
+    @tier1
+    def test_positive_end_to_end(self):
```

Comment:
Consider using a more descriptive test name here. We can assume its a positive type test (that should be in the polarion meta anyway).

This appears to be a standard 'CRUD' type test, where the given test case is creating, updating, and deleting.

Consider `test_discovery_rule_crud`

Reply:
I believe @dvagala wanted to refactor this test the old-fashioned way to maintain consistency.
He is planning to do the full transiiton to pytest test case later, so perhaps I would merge it as it is first. Would that work for you?

btw: the location of the test describes it pretty well: it's in `tests/discoveryrule.py` which reflects in the test name in the report.

Reply:
ack, its a suggestion.